src/pages/ProductDetailPage.js

```diff
@@ -0,0 +1,35 @@
+import Card from "../componnets/Card";
+import classes from "./ProductDetailsPage.module.css";
+import { useNavigate, useLoaderData } from "react-router-dom";
+
+const ProductDetailPage = ({ addProductToCart }) => {
+  const productData = useLoaderData();
+
+  const navigate = useNavigate();
+  const handleOnClick = () => {
```

Comment:
"handle" and "on" is duplication. I'd suggest
```suggestion
  const handleClick = () => {
```